Fix globals, part 4: `crypto` (#33291)
mdn/content: files/en-us/web/api/window/crypto/index.md

---
title: "Window: crypto property"
short-title: crypto
slug: Web/API/Window/crypto
page-type: web-api-instance-property
browser-compat: api.crypto
---

{{APIRef("Web Crypto API")}}

> The **`crypto`** read-only property of the {{domxref("Window")}} interface returns the {{domxref("Crypto")}} object associated to the global scope. This object allows web pages access to certain cryptographic related services.

Reviewer: ```suggestion
The **`crypto`** read-only property of the {{domxref("Window")}} interface returns the {{domxref("Crypto")}} object for this window's scope. This object gives web pages access to certain cryptographic related services.
```

```js

> globalThis.genRandomNumbers = () => {

Author: ```suggestion
window.genRandomNumbers = () => {
```

  const array = new Uint32Array(10);

>   globalThis.crypto.getRandomValues(array);

Author: ```suggestion
  window.crypto.getRandomValues(array);
```
